providers/catalog.go

```diff
@@ -115,4 +116,24 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	Keap: {
+		AuthType: Oauth2,
+		BaseURL:  "https://api.infusionsoft.com/crm/rest/v1",
```

Comment:
We want to allow users to call endpoints that may not be crm / rest / v1, so we should probably just have this as `https://api.infusionsoft.com`